packages/contracts-bedrock/contracts/libraries/Bytes.sol

```diff
@@ -131,12 +150,12 @@ library Bytes {
     /**
      * @notice Compares two byte arrays by comparing their keccak256 hashes.
      *
-     * @param _bytes First byte array to compare.
-     * @param _other Second byte array to compare.
+     * @param _a First byte array to compare.
+     * @param _b Second byte array to compare.
      *
      * @return True if the two byte arrays are equal, false otherwise.
      */
-    function equal(bytes memory _bytes, bytes memory _other) internal pure returns (bool) {
-        return keccak256(_bytes) == keccak256(_other);
+    function equal(bytes memory _a, bytes memory _b) internal pure returns (bool) {
+        return keccak256(_a) == keccak256(_b);
```

Comment:
Optimization for `equal` removed- without named return parameters, the assembly version consumes more gas.